crates/rspack_core/src/dependency/dependency_location.rs

```diff
@@ -7,31 +10,13 @@ use derivative::Derivative;
 pub struct DependencyRange {
```

Comment:
We need some comments for these types. It might be confusing to others.

Reply:
`DependencyRange` is the standardized span form used in `Dependency`, while `DependencyLocation` refers to the location of the `Dependency`. We can derive the `DependencyLocation` from the `source_map` and the `dependency_range`.